web/hand_tool.js

```diff
@@ -35,30 +35,47 @@ var GrabToPan = grabToPan.GrabToPan;
 var Preferences = preferences.Preferences;
 var SecondaryToolbar = secondaryToolbar.SecondaryToolbar;
 
-var HandTool = {
-  initialize: function handToolInitialize(options) {
-    var toggleHandTool = options.toggleHandTool;
+/**
+ * @typedef {Object} HandToolOptions
+ * @property {HTMLDivElement} container - The document container.
+ * @property {HTMLButtonElement} toggleHandTool - The button element for
+ *                                                toggling the hand tool.
+ */
+
+/**
+ * @class
+ */
+var HandTool = (function HandToolClosure() {
+  /**
+   * @constructs HandTool
+   * @param {HandToolOptions} options
+   */
+  function HandTool(options) {
+    this.container = options.container;
+    this.toggleHandTool = options.toggleHandTool;
+
```

Comment:
Since we usually add all the various members, with suitable defaults, when initializing classes, you may want to add `this.wasActive` here as well. Please refer to the `{enter,exit}PresentationMode` code.